### DISCUSSION TITLE
RQ Launcher plugin with different queues

### DISCUSSION THREAD

**Discussion by paantya** [facebookresearch/hydra#2275] [Q&A]
Can you please tell me if it is possible to set up multiple queues in the "RQ Launcher plugin"?

Let's say I want one queue for CPU servers and another for GPU servers.

**Comment by Jasha10** [comment]
Looking at the implementation of Hydra's RQ launcher, it seems that [just one instance of `rq.Queue`](https://github.com/facebookresearch/hydra/blob/8b9ce30802165c1b24dbf4e513eb78d8ca8cacd6/plugins/hydra_rq_launcher/hydra_plugins/hydra_rq_launcher/_core.py#L85) is created. This instance is used to enqueue all of the Hydra jobs that will be swept over.

**Reply by paantya** [reply]
@Jasha10  Please tell me, is it possible to make some kind of setting, thanks to which it would be possible to specify several queues?
to the primer, one pool of tasks was sent to the cpu of the server, the second task was to work on the gpu of the server, using some command: "hydra.rq.queue='cpu'" and then on different machines would raise different workers with different queue indications

**Reply by paantya** [reply]
"hydra.rq.queue='cpu'" 
and "hydra.rq.queue='gpu'" in one machine
